lib/player.js

```diff
@@ -901,7 +901,11 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     this.config_ = null;
     this.stats_ = null;
     this.videoContainer_ = null;
-    this.cmcdManager_ = null;
+
+    if (this.cmcdManager_) {
+      await this.cmcdManager_.release();
```

Comment:
You should also release the old cmcd manager, if one exists, before we call `createCmcd_()` inside the `load` method.